easybuild/easyconfigs/m/MariaDB/MariaDB-10.0.21-foss-2015b.eb

```diff
@@ -1,27 +1,24 @@
 easyblock = 'CMakeMake'
 
 name = 'MariaDB'
-version = '5.5.29'
```

Comment:
@wpoely86: no, leave the existing easyconfig, just add a new one for 10.0.21 with foss/2015b

can you look into a test report too?